autopush_rs/src/client.rs

```diff
@@ -719,8 +719,16 @@ where
 
         let now = time::precise_time_ns() / 1000;
         let elapsed = now - webpush.connected_at;
-        // XXX: tags
-        self.srv.metrics.time("ua.connection.lifespan", elapsed).ok();
+
+        let parser = Parser::new();
```

Comment:
(note that Parser's lifetime requires it here, logged: https://github.com/woothee/woothee-rust/issues/5)